.github/workflows/overview-dag.yml

```
@@ -0,0 +1,30 @@
+name: Overview DAG
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)